pkg/storaged/dialog.jsx

```diff
@@ -1188,6 +1185,21 @@ function is_expected_unmount(usage, expect_single_unmount) {
             usage[0].usage == "mounted" && usage[0].location == expect_single_unmount);
 }
 
+const teardown_block_name = use => {
+    const block_stratis = client.blocks_stratis_fsys[use.block.path];
+    const block_btrfs = client.blocks_fsys_btrfs[use.block.path];
+    let name;
+    if (block_stratis) {
+        name = block_stratis.DevNode;
+    } else if (block_btrfs) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19872-20240129-093127-4f48db75-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)